docs/development/quickstart.rst

```diff
@@ -89,9 +89,9 @@ On linux:
 
 On Windows:
 
-.. code-block:: shell
+.. code-block:: powershell
```

Comment:
I don't mind providing help for powershell, but cmd.exe must stay

Reply:
done